packages/fields/src/wordpress-editor.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@wordpress/editor';
```

Comment:
This is necessary because `@wordpress/editor` package isn't a TypeScript package